ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -3344,6 +3345,18 @@ spanning_tree:
       priority: < priority >
 ```
 
+### System Boot Settings
+
+```yaml
+boot:
+  # Set the Aboot password
+  secret:
+    # encryption_type 0, LINE -> clear_text_password (UNENCRYPTED)
+    # encryption_type 5, sha512 -> encrypted_password_string (ENCRYPTED)
+    encryption_type: < "0" | "5" | "LINE" | "sha512" >
+    password: < encrypted_password_string | clear_text_password >
```

Comment:
please remove `encryption_type`
please change `password:` to `sha512_password: "< sha_512_password >"`

Reply:
I don't agree. If we want to support multiple types, we will have multiple vars (ex. `clear_text_password`, `md5_password` and `sha512_password`), but only one of them works at a time. But we could have a default of `sha512` if that is the usual one, and then people only have to worry about password.

`LINE` is not a valid option. It is just the EOS Cli help text representation of the actual password.

Reply:
Will update.

Reply:
I will add a hierarchy. Check sha512 then md5 then clear_text password.